generation/3d_autoencoderkl/3d_autoencoderkl_tutorial.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>From this graph it looks like validation went nuts in the last epochs, is this an issue or something we should correct for here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I don't know why this is happening. This is what I meant that the performance went bonkers. Probably overfitting. Results still look OK, but if we want to prevent this we could re-run (although it takes a long time)</p>